docs/worker-node/install-singularity.md

```diff
@@ -36,14 +36,26 @@ available as at least a security update for all RHEL7-based releases.
 The document is intended for system administrators who wish to either
 install singularity or enable it to be run as an unprivileged user.
 
-!!! note "Applicable versions"
-    The applicable software versions for this document are OSG version 3.4.2 or later.
-    The version of singularity installed should be 2.3.1 or later.
+!!! note
+    There are two separate sets of instructions on this page,
+    one for [installing privileged singularity](#installing-singularity-as-privileged)
+    and one for [enabling unprivileged singularity](#enabling-unprivileged-mode-for-singularity).
 
+    As of February 2018, no VO in the OSG is ready to invoke non-setuid Singularity in production without a worker node installation.
+    Most sites will want to follow the privileged RPM install instructions until there is wider VO support.
+    You may additionally want to enable unprivileged singularity for testing.
+
+    It is possible to both install the RPM and enable unprivileged singularity, then disable the privileges in the RPM, without the VO being involved.
+    You can do that by removing the '-suid' files in the RPM and setting the singularity.conf option to disable setuid.
+    You will be responsible for removing the '-suid' files again each time the RPM is installed and merging in any changes from the RPM's singularity.conf.
```

Comment:
These instructions sound like a hack and I'm not entirely comfortable recommending this